src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ServiceRequestAPIHandler.java

```diff
@@ -59,46 +60,47 @@ private CallbackResponse makePaymentServiceReq(CallbackParams callbackParams) {
         var authToken = callbackParams.getParams().get(BEARER_TOKEN).toString();
         List<String> errors = new ArrayList<>();
         try {
-            log.info("calling payment service request {}", caseData.getCcdCaseReference());
-            String serviceRequestReference;
-
-            if (caseData.getHearingDueDate() != null) {
-                if (caseData.getHearingFeePBADetails() == null || (caseData.getHearingFeePBADetails() != null
-                    && caseData.getHearingFeePBADetails().getServiceReqReference() == null)) {
-                    serviceRequestReference = paymentsService.createServiceRequest(caseData, authToken)
-                        .getServiceRequestReference();
-
-                    caseData = caseData.toBuilder()
-                        .hearingFeePBADetails(SRPbaDetails.builder()
-                                                  .applicantsPbaAccounts(caseData.getApplicantSolicitor1PbaAccounts())
-                                                  .fee(caseData.getHearingFee())
-                                                  .serviceReqReference(serviceRequestReference).build())
-                        .build();
-                }
-            } else {
-                if (caseData.getClaimIssuedPBADetails() == null || (caseData.getClaimIssuedPBADetails() != null
-                    && caseData.getClaimIssuedPBADetails().getServiceReqReference() == null)) {
-                    serviceRequestReference = paymentsService.createServiceRequest(caseData, authToken)
-                        .getServiceRequestReference();
-
-                    caseData = caseData.toBuilder()
-                        .claimIssuedPBADetails(SRPbaDetails.builder()
-                                                   .applicantsPbaAccounts(caseData.getApplicantSolicitor1PbaAccounts())
-                                                   .fee(caseData.getClaimFee())
-                                                   .serviceReqReference(serviceRequestReference).build())
-                        .build();
-                }
+            if (isHearingFeeServiceRequest(caseData)) {
+                log.debug("Calling payment service request (hearing fee) for case {}", caseData.getCcdCaseReference());
```

Comment:
do we want these back at an info level? debug wont get picked up at the level its set currently

Reply:
We're abusing the INFO level as it is.

I made a call here to put it on DEBUG.

Reply:
are stack traces of external service calls not useful? debug would mostly be used for diagnostics scenarios, info logs are generally used in abundance, more so than debug, so that we can at a glance see successful services starting up, completing calls and get a quick overview of major or frequent state changes, 
and this call to payment service seems like something wed like to see has worked at a glance is it not?

Reply:
We can infer when something went wrong from the stack trace and the `log.error` on line 75, which is specifically around the Feign call. So, basically, we'll be able to know "at a glance" when things didn't work, which allows you to automatically infer that things did in fact work if you don't see an error.

My point is exactly that the INFO log level should not be used in abundance as it pollutes the logs and distracts away from truly informational messages. Those messages should be around configuration and the likes. Capturing the above should belong to the DEBUG level, which support teams should be able to turn on and off as needed, while TRACE should be used to get even more detailed information like which specific if/for we enter or the value of arguments (as those could disclose sensitive information).

There's multiple references online as to what should belong to different log levels but the core idea that "debug" is for troubleshooting and "info" should be used sparsely for "high level" information seems to be a constant:
https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels
https://sematext.com/blog/logging-levels/
https://www.section.io/engineering-education/how-to-choose-levels-of-logging/

As this class is essentially new and not yet in production, I'd rather start putting it on the right track by start logging things at the appropriate levels.

Reply:
After extensive consultations, other tech leads seem to prefer these things to be at `info` level. While I disagree with this assessment wholeheartedly, I'll bow to their will.

I'll be putting it back to `info` momentarily. Marking the conversation as resolved.